src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/PublishingService.cs

```diff
@@ -58,7 +63,19 @@ public async Task PublishStagedReleaseContent(Guid releaseId, string publication
             await _releaseService.SetPublishedDates(releaseId, DateTime.UtcNow);
 
             await _publicBlobCacheService.DeleteItem(new PublicationCacheKey(publicationSlug));
-            // TODO: @MarkFix EES-3149 Delete superseded publication's cache here too?
+
+            // Invalidate publication cache for superseded publications, as potentially affected. If newly
+            // published release is first Live release for the publication, the superseding is now enforced
+            var supersededPublications = await _contentDbContext.Publications
```

Comment:
Same comment as elsewhere about using the `ForEachAwaitAysnc` but just wondering if any additional testing has gone into this because in all of our automated testing and most of our manual testing we publish immediately, but this change here affects publishing with scheduled releases where the content is moved from the staging directory.

Reply:
I did manually test this at one point locally - but probably made a lot of changes since then. I'll make sure to highlight to @luke-h1 that scheduled releases also need testing with the archiving stuff.